commit 6ba1495e7c1c6b60397bdd46fd1d670c437b1422
Author: Isuru Fernando <isuruf@gmail.com>
Date:   Tue Jun 2 13:02:59 2020 -0500

    Revert "Fix issue #802: missing symbols fma/fmaf"
    
    This reverts commit 562e6cd367db6b46488984ff31f2cdfac18e8c97.

diff --git a/lib/CL/devices/common.c b/lib/CL/devices/common.c
index 256b0e15..f8345cd4 100644
--- a/lib/CL/devices/common.c
+++ b/lib/CL/devices/common.c
@@ -1342,7 +1342,7 @@ pocl_calculate_kernel_hash (cl_program program, unsigned kernel_i,
 #define DEFAULT_WG_SIZE 4096
 
 static const char *final_ld_flags[] =
-  {"-lm", "-nostartfiles", HOST_LD_FLAGS_ARRAY, NULL};
+  {"-nostartfiles", HOST_LD_FLAGS_ARRAY, NULL};
 
 static cl_device_partition_property basic_partition_properties[1] = { 0 };
 static const cl_image_format supported_image_formats[] = {
